utils/ctranslate.py

```diff
@@ -26,17 +27,25 @@ def __init__(self, model, tokenizer, compute_type):
         self.__times_invoked = 0
         self.__start_times = list()
         self.__finish_times = list()
+        self.__is_profiling = aio_profiler_enabled()
+        self.__first_run = True
 
         print("\nRunning with CTranslate2\n")
 
     def run(self, input):
-        start = time.time()
-        outputs = self.translator.translate_batch(input)
-        finish = time.time()
+        if self.__first_run:
+            self.__first_run = False
+            outputs = self.translator.translate_batch(input)
+            if self.__is_profiling:
```

Comment:
Maybe:
`if len(self._times_invoked) >= 2 and self._is_profiling:
   self.translator.init_profiling("cpu")`